### DISCUSSION TITLE
Ibxc

### DISCUSSION THREAD

**Discussion by mahfuzusal** [IBAX-io/go-ibax#200] [Ideas]
İbax (IBXC) değeri var mı şuan da bir token mı yoksa coin mi ? Bilgisi olan var mı

**Comment by byblckd** [comment]
Şuanlık herhangi bir değeri yok. Token değil coin çünkü kendi ekosistemini oluşturmayı hedefliyorlar. Eth Sol Avax gibi

**Comment by mahfuzusal** [comment]
Peki ne zaman bir değere sahip olacak ve listelenecek ? Teşekkürler!


byblckd ***@***.***> şunları yazdı (23 Oca 2022 18:50):

﻿

Şuanlık herhangi bir değeri yok. Token değil coin çünkü kendi ekosistemini oluşturmayı hedefliyorlar. Eth Sol Avax gibi

—
Reply to this email directly, view it on GitHub<https://github.com/IBAX-io/go-ibax/discussions/200#discussioncomment-2027902>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AXNXYANS7G57R4TIYGAMUWDUXQPTPANCNFSM5MTNT7YA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
